_posts/2016-01-25-drupal-6-part-3.md

```diff
@@ -0,0 +1,166 @@
+---
+layout: post
+title: "It actually is 2016, your site is running on Drupal 6, now what? Part 3: the options."
+date: 2016-01-25
+author: Chris Russo
+tags: drupal drupal8 drupal-planet 
+summary: What are your options while running a Drupal 6 site in the wild after EOL?
+featured_image: "/assets/img/options.png"
+featured_image_alt: "Options"
+drupal_planet_summary: |
+  Part 3 of a series investigating what options you have with your Drupal 6 site 
+  as EOL approaches. We discuss what your options really are given the many
+  difficult decisions you are likely faced given your organization's budget,
+  level of risk and tolerance for it, and other business priorities.
+---
+
+This is part 3 of a series investigating what to do with your Drupal 6 site as 
+EOL approaches. 
+
+([Part 1 - overview](/2015/11/24/drupal-6-upgrade.html) \| 
+[Part 2 - the risks](/2015/12/10/drupal-6-part-2.html) 
+\| **Part 3 - the options**)
+
+*****
+
+
+## The options of operating Drupal 6 after EOL
+
+<img src="/assets/img/options.png" alt="options">
+
+Though some advisers (solicited or not) might tell you otherwise, you _always_ have options as to what to do
+with your website if you're running Drupal. One of them is of course nothing, and the others require more thought.
+
+What is the right thing to do can be a hot-button topic. Some technologists are purists, 
+and believe it sacrosanct to even consider running software beyond EOL. However, we appreciate
+that although ideally site owners would always have been prepared and budgeted for 
+software upgrades, sometimes life happens, and other business priorities and budget
+availability don't perfectly align with timing for a software upgrade.
+
```

Comment:
@chrisarusso - I thought this sentence was a little lengthy and difficult to follow. Maybe there's a better way of wording it.